@types/gql.d.ts

```diff
@@ -50,6 +50,7 @@ type ProjectQuery = {
     number: string;
```

Comment:
You added `number: number` on line 53 but should also remove `number: string` here.

Reply:
Deleted!